sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -42,208 +158,269 @@
+    public Mono<Response<AttestationOpenIdMetadata>> getOpenIdMetadataWithResponse() {
+        return withContext(context -> getOpenIdMetadataWithResponse(context));
     }
 
-    /**
-     * Retrieves metadata about the attestation signing keys in use by the attestation service.
-     *
-     * @param context - Context for the operation
-     * @throws CloudErrorException thrown if the request is rejected by server.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return any object.
-     */
-    public Mono<Response<Object>> getOpenIdMetadataWithResponse(Context context) {
-        return this.metadataImpl.getWithResponseAsync(context);
+    Mono<Response<AttestationOpenIdMetadata>> getOpenIdMetadataWithResponse(Context context) {
+        return this.metadataImpl.getWithResponseAsync(context)
+            .map(generated -> Utilities.generateResponseFromModelType(generated, AttestationOpenIdMetadataImpl.fromGenerated(generated.getValue())));
     }
 
     /**
      * Retrieves metadata about the attestation signing keys in use by the attestation service.
+
+     * <p><strong>Retrieve the OpenID metadata for this async client.</strong></p>
+     * {@codesnippet com.azure.security.attestation.AttestationAsyncClient.getOpenIdMetadata}
      *
-     * @throws CloudErrorException thrown if the request is rejected by server.
+     * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return any object.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Object> getOpenIdMetadata() {
+    public Mono<AttestationOpenIdMetadata> getOpenIdMetadata() {
         // Forward the getOpenIdMetadata to the getOpenIdMetadataWithResponse API implementation.
-        return withContext(context -> this.getOpenIdMetadataWithResponse(context))
+        return this.getOpenIdMetadataWithResponse()
             .flatMap(FluxUtil::toMono);
     }
 
     /**
      * Retrieves the list of {@link AttestationSigner} objects associated with this attestation instance.
-     * An {@link AttestationSigner} represents an X.509 certificate chain and KeyId which can be used
-     * to validate an attestation token returned by the service.
+     * <p>
+     *  An {@link AttestationSigner} represents an X.509 certificate chain and KeyId which can be used
+     *  to validate an attestation token returned by the service.
+     * </p>
+     * <p><strong>Retrieve Attestation Signers for this async client.</strong></p>
+     * {@codesnippet com.azure.security.attestation.AttestationAsyncClient.getAttestationSigners}
      *
      * @return Returns an array of {@link AttestationSigner} objects.
      */
-
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AttestationSigner[]> getAttestationSigners() {
```

Comment:
>r[] [](http://example.com/codeflow?start=32&length=3)

We prefer the use of List. Also, prefer to name the methods list<items> vs get<items>

Reply:
Fixed.